### ISSUE TITLE
when faucet is disabled, interrupt user input

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/ph-scale#174]
For phetsims/QA#514 and phetsims/QA#515. @KatieWoe demonstrated this for me via Zoom. 

When either faucet becomes disabled (because the beaker is full or empty), then scenery user input needs to be interrupted by calling `interruptSubtreeInput`.    Currently, the interaction continues, and you can touch snag other things (dropper, pH probe) after the faucet has been disabled.

@KatieWoe please add the video that you captured.

**Comment by KatieWoe** [comment]
This recording is of the dropper being picked up after the faucet is disabled:
![pickupwaterdrop](https://user-images.githubusercontent.com/41024075/88090470-e0ef0280-cb4a-11ea-8885-82c084674cc5.gif)

**Comment by pixelzoom** [comment]
Reproduced on my iPad6 running iOS 13.15.1.

What's odd is that FaucetNode.js appears to be attempting to handle this:

```js
const enabledObserver = function( enabled ) {
    if ( !enabled && dragListener.isPressed ) {
      dragListener.interrupt();
    }
    if ( !enabled && tapToDispenseIsRunning ) {
      endTapToDispense();
    }
  };
  enabledProperty.link( enabledObserver );
```

**Comment by pixelzoom** [comment]
I discussed with @jonathanolson.  This is behaving as designed.  When a listener is interrupted, the pointer becomes available to begin a new Interaction -- in this case, a new drag that starts with a touch snag.   

I then discussed with @KatieWoe, and she was fine with the current behavior.  If a situation ever comes up where this behavior is not OK, she'll create a scenery issues.

So nothing to do, and closing this issue.
